README.md

```diff
@@ -305,6 +305,60 @@ If you encounter any issues with the Kwok Operator, please check the following:
   https://kwok.sigs.k8s.io/docs/user/kwok-in-cluster/
 - Check the logs of the Kwok Operator pod for any error messages under namespace kwok-operaotr.
 
+From version 1.0.0 the Kwok Operator is able to manage Statefuleset include PVC on top the nodes you provisioned above, follow these steps:
```

Comment:
fix the syntax.
From version 1.0.0 the Kwok Operator is able to manage Statefuleset. To include PVC on top of the nodes you havr provisioned above, follow these steps:

Reply:
Done